proto/access.proto

```diff
@@ -79,6 +79,12 @@ message AccessAttempt {
   // Actor describes the entity attempting to gain access.
   // Optional.
   Actor actor = 3;
+
+  // The time the last successful access attempt was made.
+  google.protobuf.Timestamp last_successful_access_time = 4;
```

Comment:
Better, I had no idea this was about successful times only. `Grant` uses the term `GRANTED`, it would be good use the same terminology here.